src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -5957,6 +5957,12 @@
                   "_meta": {
                     "description": "The interval in miliseconds between health check requests Kibana sends to the Elasticsearch."
                   }
+                },
+                "principal": {
+                  "type": "keyword",
+                  "_meta": {
+                    "description": "Indicates what elasticsearch user or service account is configured, if any."
```

Comment:
nit: It would help us a lot if we knew what was grouped under `other` without potentially exposing too much information in the payload. However, IIRC we strip the `description` out from the usage data before shipping it off to the telemetry cluster so it should be safe enough to add a little more in the description.

Reply:
Okay, I'll change this description to:

_Indicates how Kibana authenticates itself to Elasticsearch. If elasticsearch.username is configured, this can be any of: "elastic_user", "kibana_user", "kibana_system_user", or "other_user". Otherwise, if elasticsearch.serviceAccountToken is configured, this will be "kibana_service_account". Otherwise, this value will be "unknown", because some other principal might be used to authenticate Kibana to Elasticsearch (such as an x509 certificate), or authentication may be skipped altogether._

I think "unknown" is better than "other", plus it's less likely to be confused with "other_user".